src/hooks-worker-client.coffee

```diff
@@ -126,21 +126,28 @@ class HooksWorkerClient
       # If the GOBIN environment variable is set, commands are installed to the
       # directory it names instead of DIR/bin. GOBIN must be an absolute path."
       # Use the gobin if provided, otherwise fall back to gopath
-      gobin = process.env.GOBIN
-      if !gobin
-        gobin = "#{process.env.GOPATH}/bin"
-
-      @handlerCommand = "#{gobin}/goodman"
-      @handlerCommandArgs = []
-      unless which.which @handlerCommand
-        msg = '''\
-          Go hooks handler command not found in $GOBIN or $GOPATH/bin
-          Install go hooks handler by running:
-          $ go get github.com/snikch/goodman/cmd/goodman
-        '''
-        return callback(new Error(msg))
-      else
-        callback()
+      {exec} = require 'child_process'
+      getGobin = (callback) ->
+        gobin = process.env.GOBIN
+        if !gobin
+          if process.env.GOPATH
+            gobin = "#{process.env.GOPATH}/bin"
+          else
+            exec "go env GOPATH", (error, stdout, stderr) -> callback "#{stdout.trim()}/bin"
```

Comment:
I don't like the subprocess call that much. Also, I'm not sure whether this would work correctly on Windows. If Go spec clearly states the default value is `$HOME/go`, I think we can do something like

```coffee
os = require('os')

...

else
  gobin = "#{os.homedir()}/go/bin"
```

What do you think? This would also be synchronous and we wouldn't need to wrap it in the `getGobin` function.

Also note that typically, `require` statements appear on the top of each CoffeeScript (or JavaScript) file. It's not very usual to do conditional requiring.